src/ViewHelpers/ConstantViewHelper.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file belongs to the package "TYPO3 Fluid".
+ * See LICENSE.txt that was shipped with this package.
+ */
+
+namespace TYPO3Fluid\Fluid\ViewHelpers;
+
+use TYPO3Fluid\Fluid\Core\Rendering\RenderingContextInterface;
+use TYPO3Fluid\Fluid\Core\ViewHelper\AbstractViewHelper;
+use TYPO3Fluid\Fluid\Core\ViewHelper\Traits\CompileWithRenderStatic;
+
+/**
+ * Wrapper for PHPs :php:`constant` function.
+ * See https://www.php.net/manual/function.constant.php.
+ *
+ * Examples
+ * ========
+ *
+ * Get built-in PHP constant
+ * -------------------------
+ *
+ * ::
+ *
+ *    {f:constant(value: 'PHP_INT_MAX')}
+ *
+ * Output::
+ *
+ *    9223372036854775807
+ *    (Depending on CPU architecture).
+ *
+ * Get class constant
+ * ------------------
+ *
+ * ::
+ *
+ *    {f:constant(value: '\Vendor\Package\Class::CONSTANT')}
+ *
+ * Get enum value
+ * --------------
+ *
+ * ::
+ *
+ *    {f:constant(value: '\Vendor\Package\Enum::CASE')}
+ */
+class ConstantViewHelper extends AbstractViewHelper
+{
+    use CompileWithRenderStatic;
+
+    public function initializeArguments(): void
+    {
+        parent::initializeArguments();
+        $this->registerArgument('value', 'string', 'String representation of a PHP constant or enum');
```

Comment:
I'd suggest to use the parameter name `name` instead, as you specify the name of a constant to get its value as a result.